### ISSUE TITLE
ARM64 build?

### ISSUE THREAD

**Issue by cdkw2** [SheerSt/pokewilds#429]
Any future plans to make a arm64 build for this? Would love to play this on my raspberry pi.

**Comment by fourlastor** [comment]
You can! You need to download the `otherplatforms` build and install java on your machine, for example via https://sdkman.io

```bash
curl -s "https://get.sdkman.io" | bash
source "$HOME/.sdkman/bin/sdkman-init.sh"
sdk install java
```

when you get prompted:
> `Do you want java 17.0.7-tem to be set as default? (Y/n):`

input `y` and press enter.

then you can play by reading the `README.txt` file and referring to the Linux instructions

**Comment by cdkw2** [comment]
@fourlastor I got it to work! I even made a little shortcut for it in the accessories tab!

![pokewilds](https://github.com/SheerSt/pokewilds/assets/67598073/a00c9f77-5309-4cac-a640-d457ee8e5586)

**Comment by cdkw2** [comment]
It launches in like 2 seconds!

![PokeWilds](https://i.imgur.com/NKf6oxD.jpg)

**Comment by fourlastor** [comment]
That's super nice to hear! Be mindful of the world size, right now pokewilds is very memory hungry

**Comment by cdkw2** [comment]
> That's super nice to hear! Be mindful of the world size, right now pokewilds is very memory hungry

@fourlastor 

Yeah it is definitely ram hungry. I tried making a world on XXL size and it took a solid 34 minutes to generate. I am on a 4GB ram version for the context and I had nothing else running in the background, so almost all of the ram was dedicated on the world generation. Although, after generating the world it ran smoothly and ram consumption was also low. I was able to play the game and listed to music in background with less than 50% ram usage. Saving the world takes less than 10 seconds and loading the XXL world from saves takes about 5 minutes. It increases as parts of the map gets discovered. So yeah, if you are a patient person it's pretty good. I am patient so I can play with ease. I hope this comment helps you in your future optimization, if you want to test anything for a raspberry pi, let me know I would be more than happy :)
